src/components/Dashboard/TodoList.vue

```diff
@@ -445,17 +470,17 @@ export default {
   methods: {
     ...mapActions(['setARFilingYear', 'setCurrentFilingStatus']),
 
-    loadData () {
+    async loadData () {
```

Comment:
I'm not 100% sure if you need this async, when you've applied it to the scope of the callback below?

Reply:
you are correct! removed